File: app/src/main/java/com/codefororlando/orlandowalkingtours/models/Location.java
package com.codefororlando.orlandowalkingtours.models;

import java.io.Serializable;

/**
 * Created by MarkoPhillipMarkovic on 5/9/2016.
 */
//This class holds location data. Use it for mapping data from the datasource.
public class Location implements Serializable {
    private String mType;
    private double mLatitude;
    private double mLongitude;

    public Location(String type, double latitude, double longitude){
        mType = type;
        mLatitude = latitude;
        mLongitude = longitude;
    }


    public String getType() {
        return mType;
    }

    public double getLatitude() {
        return mLatitude;
    }

    public double getLongitude() {
        return mLongitude;
    }
}

File: app/src/main/java/com/codefororlando/orlandowalkingtours/entities/Location.kt
package com.codefororlando.orlandowalkingtours.entities

/**
 * Created by ryan on 10/12/17.
 */
data class Location(val name: String,
                    val address: String,
                    val coordinates: Coordinates,
                    val pictures: List<String>,
                    val registryDates: RegistryDates)